src/BinSkim.Sdk/BinaryAnalyzerContext.cs

```diff
@@ -74,6 +74,7 @@ public string MimeType
         }
 
         public RuntimeConditions RuntimeErrors { get; set; }
+        public bool AnalysisComplete { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }

```

Comment:
Should I change to `{get;set;}`?
How should I use this?

Reply:
Yes, change to { get; set; } as you say. :)

---
In reply to: [521325596](https://github.com/microsoft/binskim/pull/300#discussion_r521325596) [](ancestors = 521325596)